src/shared/cryptsetup-fido2.c

```diff
@@ -247,12 +243,12 @@ int acquire_fido2_key_auto(
                         required |= FIDO2ENROLL_UV_OMIT; /* compat with 248 */
 
                 ret = acquire_fido2_key(
-                                name,
-                                friendly_name,
+                                NULL,
```

Comment:
mark the NULL/0/true/false parameters with their names - ie, `/* volume_name= */ NULL,` here and elsewhere

Reply:
Added comments for argument names, but I didn't change the ones in `prepare_luks_keyfile()` because code there is copied as-is from the original `prepare_luks()` function.